sashimi/gui/camera_gui.py

```diff
@@ -296,8 +285,9 @@ def set_full_frame(self):
     def update_roi_info(self):
         dims = self.state.camera_settings.roi
         binning = convert_camera_params(self.state.camera_settings).binning
-        self.lbl_roi.setText(
-            f"Current frame dimensions are:\nHeight: {int(dims[0] / binning)}\nWidth: {int(dims[1] / binning)}"
+
+        self.status_bar.addMessage(
```

Comment:
Having these appearing and disappearing messages in random places will be quite unwieldy. I also don't think the status bar should be an attribute of the CameraSettings. Rather, the StatusBar can be a subclass of [StatusBar](https://doc.qt.io/qt-5/qstatusbar.html) that in turn has widgets (just labels) that keep track of the state.